.github/workflows/boot-tests-nightly.yml

```
@@ -72,20 +72,27 @@ jobs:
         id: playwright-tests
         env:
           BASE_URL: https://stage.foo.redhat.com:1337
+          GLOBAL_STATIC_USER: ${{ vars.GLOBAL_STATIC_USER }}
         run: |
           export PLAYWRIGHT_USER=image-builder-playwright-$RANDOM
           export PLAYWRIGHT_PASSWORD=image-builder-playwright-$(uuidgen)
-          # Step 1: Create a new empty account
-          curl -k -X POST https://account-manager-stage.app.eng.rdu2.redhat.com/account/new -d "{\"username\": \"$PLAYWRIGHT_USER\", \"password\":\"$PLAYWRIGHT_PASSWORD\"}"
-          # Step 2: Attach subscriptions to the new account
-          curl -k -X POST https://account-manager-stage.app.eng.rdu2.redhat.com/account/attach \
-            -d "{\"username\": \"$PLAYWRIGHT_USER\", \"password\":\"$PLAYWRIGHT_PASSWORD\", \"sku\":[\"RH00003\"],\"quantity\": 1}"
-          # Step 3: Activate the new account by accepting Terms and Conditions
-          curl -k -X POST https://account-manager-stage.app.eng.rdu2.redhat.com/account/activate -d "{\"username\": \"$PLAYWRIGHT_USER\", \"password\":\"$PLAYWRIGHT_PASSWORD\"}"
-          # Step 4: Refresh account to update subscription pools
-          curl -k -X POST https://account-manager-stage.app.eng.rdu2.redhat.com/account/refresh -d "{\"username\": \"$PLAYWRIGHT_USER\", \"password\":\"$PLAYWRIGHT_PASSWORD\"}"
-          # Step 5: View account to check account status
-          curl -k -X GET "https://account-manager-stage.app.eng.rdu2.redhat.com/account/get?username=$PLAYWRIGHT_USER&password=$PLAYWRIGHT_PASSWORD"
+          export GLOBAL_STATIC_USER=$GLOBAL_STATIC_USER
```

```suggestion
          echo "Global statis user before exporting $GLOBAL_STATIC_USER"
          export GLOBAL_STATIC_USER=$GLOBAL_STATIC_USER
```
Can you try this? If this is not empty, I'd try not using the same variable name here on the left and on the right? But this is just a wild guess.